content/blog/2018/06/2018-06-15-anchore-image-scanning.adoc

```diff
@@ -0,0 +1,173 @@
+
+
+While some may feel that “fail fast” sounds rather negative (especially regarding security), you could better describe this process as “learn fast” as mistakes are found earlier in the development cycle and can be easily corrected.
+
+
+The increased use of CI/CD platforms such as Jenkins has helped to improve the efficiency of development teams and streamlined the testing process and we can leverage the same CI/CD infrastructure to improve the security of our container deployments.
+
+For many organizations the last step before deploying an application is for the security team to perform an audit, this may entail scanning the image for vulnerable software components, for example outdated operating system packages that contain known security vulnerabilities (CVEs), to verify that the applications and operating systems are correctly configured and that the organization’s best practices and compliance policies have been correctly implemented.
+
+
+In this post we walk through adding security and compliance checking into the CI/CD process so you can “learn fast” and correct any security or compliance issues early in the development cycle.
+
+
+This document will outline the steps to deploy Anchore’s open source security and compliance scanning engine with Jenkins to add analytics, compliance and governance to your CI/CD pipeline.
+
+
+Anchore has been designed to plug seamlessly into the CI/CD workflow, where a developer commits code into the source code management system, which then triggers Jenkins to start a build that creates a container image. In the typical workflow this container image is then run through automated testing. If an image does not meet your organization’s requirements for security or compliance then it makes little sense to invest the time required to perform automated tests on the image, it would be better to “learn fast” by failing the build and returning the appropriate reports back to the developer to allow the issue to be addressed.
+
+image:/images/post-images/2018-06-15-anchore/anchore-flow.png[title="Anchore process", role="center", width=600]
+
+
+Anchore has published a link:https://plugins.jenkins.io/anchore-container-scanner[plugin] for Jenkins which, along with Anchore’s open source engine or Enterprise offering, allows container analysis and governance to be added quickly into the CI/CD process.
+
+The following guide will allow you to add image scanning and analysis into your CI/CD process in less time than it has already taken to read this blog post!
```

Comment:
There are several places in this post where we tell you what we are about to tell you. I leave it to your discretion as to whether you want to leave that in, to me it's just a bit superfluous.